### ISSUE TITLE
Missing notifications investigations

### ISSUE THREAD

**Issue by Tyuoli** [vector-im/riot-android#2107]
Did you notice the absence of notifications ?

**Comment by neverfox** [comment]
Yes. I'm actually just starting to use the Android app today, so I was wondering if I was actually supposed to see any or not. The settings are ambiguous about whether or not they're sound notifications only, but sound is all I'm getting, in any case. I'm also not sure if it has anything to do with conversations being e2e encrypted (i.e. it's some kind of security measure), but without visible notifications, I'm going to have a hard time using this as a primary tool.

**Comment by zwieberl** [comment]
Yes. Testing in a group of multiple people and multiple devices (different android versions, etc.) at the moment. 
Some have no problems, others sporadically miss notifications, while others seem to never get any notifications.
This has been a big item for criticism and might be a roadblock to switching to riot/matrix completely.
